package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@holochain/client",
-  "version": "0.18.0-rc.2",
+  "version": "0.18.0",
```

Comment:
package.lock should get updated when this changes?

Reply:
indeed...updated now.